ROADMAP.md

```diff
@@ -0,0 +1,9 @@
+# Chaos Mesh Roadmap
+
+This document defines the roadmap for Chaos Mesh development.
+
+## v0.1.0
+- [ ] Support bpf injection for time skew chaos
+- [ ] Add container kill/failure chaos
+- [ ] Add cpu chaos
+- [ ] Add memory chaos
```

Comment:
Make scheduler optional？

Reply:
What's the difference between pod failure and container failure?

Reply:
can we provides more detailed information about each chaos type and let community know the detail of each item.

Reply:
we will add detail info in issues